### PR TITLE
docs(elastic-metal): add how-to run kubernetes nodes on RISC-V int-add-elastic-metal

### PR DESCRIPTION
**Pull request by Thezap**
### Your checklist for this pull request

- [ X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

Add an how to to run kubernetes nodes on RISC-V using the new EM-RV1 server and Kosmos

### CODE REVIEW

**Review by ofranc — APPROVED** [review]

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+meta:
+  title: How to run kubernetes nodes on RISC-V?
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:3]:
```suggestion
  title: How to run Kubernetes nodes on RISC-V
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  description: This page explains how to used your EM-RV1 Elastic Metal server as Kosmos Kubernetes node
+content:
+  h1: How to run kubernetes nodes on RISC-V?
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:6]:
```suggestion
  h1: How to run Kubernetes nodes on RISC-V
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    export POOL_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    export POOL_REGION=xx-xxx
+    export SCW_SECRET_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    ```
+
+8. Run node agent
+
+    `sudo -E ./node-agent_linux_riscv64 -loglevel 0 -no-controller && sleep 10`
+
+9. **Restart** your server
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:45]:
```suggestion
9. **Restart** your server.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+5. Connect via SSH to your Elastic Metal RV1 server
+6. Export noted env variables and secret key
+
+    ```
+    export POOL_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    export POOL_REGION=xx-xxx
+    export SCW_SECRET_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    ```
+
+8. Run node agent
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:41]:
```suggestion
8. Run node agent.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+If you don't have an Elastic Metal RV1, you can follow [this guide](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/create-server/).  
+Note that Elastic Metal RV1s are only available in the `FR-PAR-2` region in the `Labs` section.
+
+
+1. [Install the server](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/install-server/) with the `Kosmos` image.
+2. Go to the **Nodes** tab of your Kosmos cluster and click **Add external node**.
+3. Do not follow the instructions on the page to connect your server into the pool.  
+Note only the `POOL_ID` and `POOL_REGION`.
+
+4. Create a Scaleway secret key with the `KubernetesExternalNodeRegister` IAM permission set. [How to create a secret key](https://www.scaleway.com/docs/console/my-project/how-to/generate-api-key/)
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:31]:
```suggestion
4. Create a Scaleway secret key with the `KubernetesExternalNodeRegister` IAM permission set. [How to create a secret key](https://www.scaleway.com/docs/console/my-project/how-to/generate-api-key/).
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+5. Connect via SSH to your Elastic Metal RV1 server
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:32]:
```suggestion
5. Connect via SSH to your Elastic Metal RV1 server.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+6. Export noted env variables and secret key
```
Comment (ldecarvalho-doc) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:33]:
```suggestion
6. Export noted env variables and secret key.
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+RISC-V is not yet officially supported in Kubernetes, but to let you test this technology with kube containers, we've prepared an image with the necessary pre-built binaries.
+
+Thanks to Scaleway Kosmos and our Elastic Metal RV1, you can follow this guide to launch RISC-V containers on Kubernetes effortlessly and in just a few minutes.
+
+All you need is a Kosmos cluster (currently only version 1.29.1), configured with a pool of external nodes and an Elastic Metal RV1 server.
+
+If you don't have a Kosmos cluster, you can follow [this guide](https://www.scaleway.com/en/docs/containers/kubernetes/how-to/create-kosmos-cluster/) to create your first Kosmos cluster.
+
+If you don't have an Elastic Metal RV1, you can follow [this guide](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/create-server/).
```
Comment (Thezap) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:22]:
```suggestion
If you don't have an Elastic Metal RV1, you can follow [this procedure](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/create-server/).  
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+tags: kosmos riscv elastic-metal
+dates:
+categories:
+  - bare-metal
+---
+
+RISC-V is not yet officially supported in Kubernetes, but to let you test this technology with Kube containers, we have prepared an image with the necessary pre-built binaries.
+
+Thanks to Scaleway Kosmos and our Elastic Metal RV1, you can follow this guide to launch RISC-V containers on Kubernetes effortlessly and in just a few minutes.
+
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:17]:
```suggestion

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
- A [Kubernetes Kosmos cluster](/containers/kubernetes/how-to/create-cluster) version 1.29.1 with a pool of external nodes
- An [Elastic Metal](/bare-metal/elastic-metal/quickstart/) RV1 server
    Elastic Metal RV1 servers are only available in the `FR-PAR-2` region, in the `Labs` section.

```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+1. [Install the server](https://www.scaleway.com/en/docs/bare-metal/elastic-metal/how-to/install-server/) with the `Kosmos` image.
+2. Go to the **Nodes** tab of your Kosmos cluster and click **Add external node**.
+3. Do not follow the instructions on the page to connect your server into the pool.  
+Note only the `POOL_ID` and `POOL_REGION`.
+
+4. Create a Scaleway secret key with the `KubernetesExternalNodeRegister` IAM permission set. [How to create a secret key](https://www.scaleway.com/docs/console/my-project/how-to/generate-api-key/).
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:23]: As we've added this as a requirement, we ca delete step 4.

Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:23]:
```suggestion
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+5. Connect via SSH to your Elastic Metal RV1 server.
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:24]:
```suggestion
4. Connect via SSH to your Elastic Metal RV1 server.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+6. Export noted env variables and secret key.
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:25]:
```suggestion
5. Export noted env variables and secret key.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+
+    ```
+    export POOL_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    export POOL_REGION=xx-xxx
+    export SCW_SECRET_KEY=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+    ```
+
+8. Run node agent.
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:33]:
```suggestion
6. Run node agent.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+
+    `sudo -E ./node-agent_linux_riscv64 -loglevel 0 -no-controller && sleep 10`
+
+9. **Restart** your server.
```
Comment (jcirinosclwy) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:37]:
```suggestion
7. **Restart** your server.
```

File: bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx [hunk 15]
```diff
@@ -0,0 +1,37 @@
+---
+meta:
+  title: How to run Kubernetes nodes on RISC-V
+  description: This page explains how to used your EM-RV1 Elastic Metal server as Kosmos Kubernetes node
+content:
+  h1: How to run Kubernetes nodes on RISC-V
+  paragraph:  This page explains how to used your EM-RV1 Elastic Metal server as Kosmos Kubernetes node
+tags: kosmos riscv elastic-metal
+dates:
```
Comment (nerda-codes) [inline bare-metal/elastic-metal/how-to/kubernetes-on-riscv.mdx:9]:
```suggestion
dates:
  validation: 2024-03-22
  posted: 2024-03-22
```

**Review by nerda-codes — APPROVED** [review]

**Review by ofranc — APPROVED** [review]

**Pull request merged**